projects/admin/src/app/classes/library.ts

```diff
@@ -137,21 +137,31 @@ export class Library {
   // CONSTRUCTOR ===================================================
   /**
    * Constructor
-   * @param obj: the metadata to use to create a Library object.
+   * @param obj - the metadata to use to create a Library object.
    */
   constructor(obj?: any) {
     this._initOpeningHours();
     if (obj) {
       this.update(obj);
     }
-    if (!this.exception_dates) {
+    if (!this.exception_dates) {
```

Comment:
I do not see the changes.

Reply:
it was a non-breakable-space, replace with a simple space.